package/rules/file.js

```diff
@@ -3,7 +3,17 @@ const { source_path } = require('../config')
 const assetHost = require('../asset_host')
 
 module.exports = {
-  test: /\.(jpg|jpeg|png|gif|tiff|ico|svg|eot|otf|ttf|woff|woff2)$/i,
+  exclude: [
+    /\.(js|jsx)?(\.erb)?$/,
+    /\.coffee(\.erb)?$/,
+    /\.(ts|tsx)?(\.erb)?$/,
+    /\.vue(\.erb)?$/,
+    /\.elm(\.erb)?$/,
+    /\.(scss|sass)$/i,
+    /\.(css)$/i,
+    /\.html$/,
+    /\.json$/
+  ],
```

Comment:
Looks like this could be combined to a single regex.

Reply:
Yep, will do 👍